README.md

```diff
@@ -1 +1,162 @@
 # group4msauto
+## Introduction
+
+FinalProject is an automation project for testing Asset Management
+
+## Project Structure
+
+The project is organized as the structure below
+
+- **Data**: Includes data driven files
+- **Helpers**: 
+  - Automation_Test_Framework contains all base methods of pages
+  - Core_Framework contains all base methods of test/driver interaction
+- **Pages**: Includes pages
+  - HomePage.cs
+  - LoginPage.cs
+  - ManageAssetPage.cs
+  - ManageAssignmentPage.cs
+  - ManageUserPage.cs
+  - So on...
+- **Results**: Includes test results/reports after executing tests 
+- **Tests**: Includes all tests scripts divided by features
+  - APITest.cs
+  - LoginTest.cs
+  - LogoutTest.cs
+  - ManageAssetTest.cs
+  - ManageAssignmentTest.cs
+  - ManageUserTest.cs
+
+
+
+## Coding Convention
+
+Examples for pages
+```cs
+WebDriverBase.cs
+// Settings
+using Core_Framework.Reporter;
+
+// Variables 
+public IWebDriver Driver;
+public IJavaScriptExecutor Javascript { get; set; }
+private WebDriverWait _explicitWait;
+private Actions _actions;
+private int _timeWait = 60;
+
+// Methods
+public void MoveForward()
+{
```

Comment:
This is not the page sample. You should write by yourself. Don't try to copy 100% of the idea